demos/serverless/src/handlers.js

```diff
@@ -61,21 +70,28 @@ exports.join = async(event, context) => {
       // For simplicity here, we use the meeting title.
       ExternalMeetingId: query.title.substring(0, 64),
 
-      // Tags associated with the meeting. They can be used in cost allocation console
-      Tags: [
-        { Key: 'Department', Value: 'RND'}
-      ]
     };
-    console.info('Creating new meeting: ' + JSON.stringify(request));
-    meeting = await chime.createMeeting(request).promise();
+
+    if (useGlobalEndpoint) {
+      request = {
+        ...request,
+        // Tags associated with the meeting. They can be used in cost allocation console
+        //Tags are not supported by new ChimeMeetingsSDK
```

Comment:
"Tags are not supported by the new ChimeSDKMeetings namespace"

Reply:
yes but demo currently utilizes Tags for global endpoints so i have used tags based upon controlRegion.

Reply:
Sure, I'm saying the comment needs to change: the namespace is "ChimeSDKMeetings"

Reply:
ohh. Got it

Reply:
updated